tests/unit/core/model/entity/test_base.py

```diff
@@ -43,3 +47,28 @@ def test_smart_weight_loading(self, mocker) -> None:
             current_model.state_dict()["model.head.bias"].index_select(0, indices)
             == prev_model.state_dict()["model.head.bias"],
         )
+
+
+class TestOVModel:
+    @pytest.fixture()
+    def entity(self) -> OTXBatchDataEntity:
```

Comment:
how about?
```suggestion
    def batch(self) -> OTXBatchDataEntity:
```
 or `dataset_batch` or `input_batch`, but not `entity` :)

Reply:
the reason is entity is not clear and too generic. In the future, I'll propose removing `Entity` from all of these objects :)

Reply:
input_batch looks good for me

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2769/commits/0c846b7d0e0e44feaf1f1e42bfd5202b122102a3)